src/components/core/PivotTable.tsx

```diff
@@ -216,6 +216,8 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
     private lastResizedHeight = 0;
     private numberOfColumnResizedCalls = 0;
     private waitingForFirstExecution = true;
+    private isKeyMetaPressed = false;
```

Comment:
I would prefer both key modifiers in a single variable, as it does not make sense to be both true at the same time.

Reply:
done